include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -12,10 +12,9 @@ class SoftphoneManager
 {
 public:
     SoftphoneManager(int number_of_softphones, int port, std::string domain):
-        _port(port), _domain(domain)
+        _port(std::move(port)), _domain(std::move(domain))
```

Comment:
No need to move an `int` as it is a primitive type, you can pass it by value.

Reply:
done